### DISCUSSION TITLE
[DAY4]피어세션

### DISCUSSION THREAD

**Discussion by karim-moon** [bcaitech1/p4-fr-soccer#21] [Peersession]
# Data

## Shape

앙상블 → 가로, 세로 방향 4개 모델을 각각 따로 학습시킨다음에 앙상블을 하면 어떨까?

분수 → 분수는 가로방향으로 Sequence 데이터라고 말할 수 있을까? 세로로 학습시켜도 좋을것같다

input 라벨 추가 → 이미지가 가로인지, 세로인지 알려주는 레이블을 추가

## Fold

베이스라인은 random split으로 데이터셋이 나누어짐

정환님 → k-fold로 나눈 코드 PR

## 사용되지않는 source.txt

Baseline에는  gt.txt만 사용되고 source.txt는 사용되지 않음.

level정보가 들어가있는 souce.txt를 어떻게 사용할 수 있을까?

## Vocab

tokens.txt는 train 데이터에 있는 정보를 담고있다

하지만 모든 LaTex vocab이 들어있지는 않음 → 이미지에 현재 tokens.txt에 없는 정보를 concat하고 새로운 vocab을 적용하면 되지 않을까?

# 실험

SATRN → 1 epoch 50분 소요 

최대한 실험시간을 단축하고 여러번 실험하여 가장 좋은 결과를 내는 모델을 사용하는게 좋을것같다

## 방법1

전체 데이터를 1/10로 쪼개서 학습시킨다. (split_gt함수를 overloading하는 방식으로 진행)

## 방법2

학습 이미지 사이즈를 줄인다

## 방법3

학습 레이어를 줄인다.

**Comment by bellamy-k** [comment]
작성을 깜빡했네요 죄성합니다 ㅠㅠ
